doc/src/records/dhwsys.md

```diff
@@ -227,6 +420,94 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
   ----------- ----------------- ------------- -------------- -----------------
     ^o^F        $>$ 32 ^o^F      wsTSetPoint        No             hourly
 
+**wsTSetpointDes=*float***
+
+Specifies the design (sizing) setpoint temperature.
+
+<%= member_table(
+  units: "^o^F",
+  legal_range: "x $>$ 32 ^o^F",
+  default: "wsTUse",
+  required: "No",
+  variability: "constant") %>
+
+**wsVolRunningDes=*float***
+
+Running volume for design.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $>$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "constant") %>
+
+**wsASHPTSrcDes=*float***
+
+Design (sizing) source air temperature for HPWH.
+
+<%= member_table(
+  units: "^o^F",
+  legal_range: "x $>$ 32 ^o^F",
+  default: "Heating design temperature",
+  required: "No",
+  variability: "At the start and at the end of interval") %>
+
+**wsFxDes=*float***
+
+Excess size factor for domestic hot water design. wsFxDes is applied when wsHeatingCapDes and/or wsVolRunningDes are defaulted from EcoSizer at the end of the prerun. There is no effect if those values are input.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $>$ 0",
+  default: "1.0",
+  required: "No",
+  variability: "constant") %>
+
+**wsDRMethod=*choice***
+
+Response control method for domestic hot water demand.
+
+<%= csv_table(<<END, :row_header => true)
+Choice, Description
+C_DHWDRMETH_NONE, None
```

Comment:
Valid user input for choices is found in CNDTYPES.DEF. Look for the choice name between the underbars (e.g., DHWDRMETH).